src/Commands/Owner/SetCommand.php

```diff
@@ -42,7 +42,7 @@ public function setOwner($site_name, $owner)
         $this->processWorkflow($site->setOwner($user->id));
         $this->log()->notice(
             'Promoted {user} to owner of {site}',
-            ['user' => $user->getName(), 'site' => $site->getName(),]
+            ['user' => $owner, 'site' => $site->getName(),]
```

Comment:
The `@param` says `$owner` could be uuid, email, and full name of the user.
So people could still use the user's full name to set owner and then display the user's full name?